### PR TITLE
OP-946: Check and graphql query - unique location code.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
Ticket: https://openimis.atlassian.net/browse/OP-946

FE PR: https://github.com/openimis/openimis-fe-location_js/pull/32

Changes:
- Added BE check for validating location code.
- Added graphql query for checking uniqueness of location code

[OP-946]: https://openimis.atlassian.net/browse/OP-946?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dborowiecki — CHANGES_REQUESTED** [review]
Please find my comments and align code with changes introduced in [OP-948](https://github.com/openimis/openimis-be-location_py/pull/44/files)

File: location/schema.py [hunk 1]
```diff
@@ -46,6 +52,13 @@ def resolve_health_facilities(self, info, **kwargs):
             query = HealthFacility.filter_queryset(query)
         return gql_optimizer.query(query.all(), info)
 
+    def resolve_validate_location_code(self, info, **kwargs):
+        errors = validate_location_code(kwargs['location_code'])
+        if errors:
+            return False
+        else:
+            return True
+
```
Comment (delcroip) [inline location/schema.py:61]:
```suggestion
            return bool(errors)
```

Comment (delcroip) [inline location/schema.py:61]:
```suggestion
        return False if errors else True

```

**Review by dborowiecki — APPROVED** [review]
Looks good.

**Pull request merged**